tss-esapi/src/context/tpm_commands/attestation_commands.rs

```diff
@@ -322,8 +322,47 @@ impl Context {
         ))
     }
 
+    /// Get the current time and clock from the TPM
+    ///
```

Comment:
It would be excellent if you could add an runable example and documentation of the parameters.